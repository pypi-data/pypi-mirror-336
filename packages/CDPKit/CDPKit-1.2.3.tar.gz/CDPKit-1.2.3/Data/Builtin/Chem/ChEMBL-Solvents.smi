WATER	[OH2]
DICHLOROMETHANE	ClCCl
TRICHLOROMETHANE	ClC(Cl)Cl
ETHYL ACETATE	CCOC(=O)C
METHANOL	CO
PROPAN-2-OL	CC(C)O
ACETONE	CC(=O)C
DMSO	CS(=O)C
ETHANOL	CCO
