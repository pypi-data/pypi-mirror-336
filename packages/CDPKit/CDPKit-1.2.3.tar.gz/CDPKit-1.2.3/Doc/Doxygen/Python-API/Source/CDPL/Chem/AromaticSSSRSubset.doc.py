#
# This file is part of the Chemical Data Processing Toolkit
#
# Copyright (C) Thomas Seidel <thomas.seidel@univie.ac.at>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; either
# version 2 of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with this program; see the file COPYING. If not, write to
# the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
# Boston, MA 02111-1307, USA.
#

##
# \brief Implements the extraction of the aromatic rings in the SSSR of a molecular graph.
# 
class AromaticSSSRSubset(FragmentList):

    ##
    # \brief Constructs an empty <tt>AromaticSSSRSubset</tt> instance.
    # 
    def __init__() -> None: pass

    ##
    # \brief Initializes a copy of the \e %AromaticSSSRSubset instance \a arom_sssr.
    # \param arom_sssr The \e %AromaticSSSRSubset instance to copy.
    # 
    def __init__(arom_sssr: AromaticSSSRSubset) -> None: pass

    ##
    # \brief Construct a <tt>AromaticSSSRSubset</tt> instance that contains all aromatic rings of the SSSR of the molecular graph <em>molgraph</em>.
    # 
    # \param molgraph The molecular graph for which to extract the aromatic rings.
    # 
    def __init__(molgraph: MolecularGraph) -> None: pass

    ##
    # \brief Replaces the current state of \a self with a copy of the state of the \c %AromaticSSSRSubset instance \a arom_sssr.
    # \param arom_sssr The \c %AromaticSSSRSubset instance to copy.
    # \return \a self
    # 
    def assign(arom_sssr: AromaticSSSRSubset) -> AromaticSSSRSubset: pass

    ##
    # \brief Replaces the current set of rings by the aromatic rings in the SSSR of the molecular graph <em>molgraph</em>.
    # 
    # \param molgraph The molecular graph for which to extract the aromatic rings.
    # 
    def extract(molgraph: MolecularGraph) -> None: pass
