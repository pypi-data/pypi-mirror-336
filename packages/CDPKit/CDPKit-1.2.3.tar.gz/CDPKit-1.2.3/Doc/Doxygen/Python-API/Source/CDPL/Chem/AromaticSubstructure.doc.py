#
# This file is part of the Chemical Data Processing Toolkit
#
# Copyright (C) Thomas Seidel <thomas.seidel@univie.ac.at>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; either
# version 2 of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with this program; see the file COPYING. If not, write to
# the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
# Boston, MA 02111-1307, USA.
#

##
# \brief Implements the perception of aromatic atoms and bonds in a molecular graph.
# 
class AromaticSubstructure(Fragment):

    ##
    # \brief Constructs an empty <tt>AromaticSubstructure</tt> instance.
    # 
    def __init__() -> None: pass

    ##
    # \brief Construct a <tt>AromaticSubstructure</tt> instance that consists of the aromatic atoms and bonds of the molecular graph <em>molgraph</em>.
    # 
    # \param molgraph The molecular graph for which to perceive aromatic atoms and bonds.
    # 
    def __init__(molgraph: MolecularGraph) -> None: pass

    ##
    # \brief Replaces the currently stored atoms and bonds by the set of aromatic atoms and bonds of the molecular graph <em>molgraph</em>.
    # 
    # \param molgraph The molecular graph for which to perceive aromatic atoms and bonds.
    # 
    def perceive(molgraph: MolecularGraph) -> None: pass
