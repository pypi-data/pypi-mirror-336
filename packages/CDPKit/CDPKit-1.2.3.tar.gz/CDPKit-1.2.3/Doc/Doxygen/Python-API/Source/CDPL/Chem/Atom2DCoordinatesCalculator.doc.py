#
# This file is part of the Chemical Data Processing Toolkit
#
# Copyright (C) Thomas Seidel <thomas.seidel@univie.ac.at>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; either
# version 2 of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with this program; see the file COPYING. If not, write to
# the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
# Boston, MA 02111-1307, USA.
#

##
# \brief Atom2DCoordinatesCalculator.
# 
class Atom2DCoordinatesCalculator(Boost.Python.instance):

    ##
    # \brief Constructs the <tt>Atom2DCoordinatesCalculator</tt> instance.
    # 
    def __init__() -> None: pass

    ##
    # \brief Constructs the <tt>Atom2DCoordinatesCalculator</tt> instance and calculates 2D-coordinates for the atoms of the molecular graph <em>molgraph</em>.
    # 
    # \param molgraph The molecular graph for which to calculate 2D-coordinates.
    # \param coords An array containing the calculated 2D-coordinates. The coordinates are stored in the same order as the atoms appear in the atom list of the molecular graph (i.e. the coordinates of an atom are accessible via its index).
    # 
    def __init__(molgraph: MolecularGraph, coords: Math.Vector2DArray) -> None: pass

    ##
    # \brief Returns the numeric identifier (ID) of the wrapped C++ class instance.
    # 
    # Different Python \c %Atom2DCoordinatesCalculator instances may reference the same underlying C++ class instance. The commonly used Python expression
    # <tt>a is not b</tt> thus cannot tell reliably whether the two \c %Atom2DCoordinatesCalculator instances \e a and \e b reference different C++ objects. 
    # The numeric identifier returned by this method allows to correctly implement such an identity test via the simple expression
    # <tt>a.getObjectID() != b.getObjectID()</tt>.
    # 
    # \return The numeric ID of the internally referenced C++ class instance.
    # 
    def getObjectID() -> int: pass

    ##
    # \brief Calculates 2D-coordinates for the atoms of the molecular graph <em>molgraph</em>.
    # 
    # \param molgraph The molecular graph for which to calculate 2D-coordinates.
    # \param coords An array containing the calculated 2D-coordinates. The coordinates are stored in the same order as the atoms appear in the atom list of the molecular graph (i.e. the coordinates of an atom are accessible via its index).
    # 
    def calculate(molgraph: MolecularGraph, coords: Math.Vector2DArray) -> None: pass

    objectID = property(getObjectID)
