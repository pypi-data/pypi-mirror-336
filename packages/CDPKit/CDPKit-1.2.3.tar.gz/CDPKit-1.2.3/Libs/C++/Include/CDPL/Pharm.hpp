/* 
 * Pharm.hpp 
 *
 * This file is part of the Chemical Data Processing Toolkit
 *
 * Copyright (C) 2003 Thomas Seidel <thomas.seidel@univie.ac.at>
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License
 * along with this library; see the file COPYING. If not, write to
 * the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 * Boston, MA 02111-1307, USA.
 */

/**
 * \file
 * \brief A convenience header including everything that is defined in namespace CDPL::Pharm.
 */

#ifndef CDPL_PHARM_HPP
#define CDPL_PHARM_HPP

#include "CDPL/Pharm/Feature.hpp"
#include "CDPL/Pharm/BasicFeature.hpp"
#include "CDPL/Pharm/FeatureContainer.hpp"
#include "CDPL/Pharm/FeatureSet.hpp"
#include "CDPL/Pharm/Pharmacophore.hpp"
#include "CDPL/Pharm/BasicPharmacophore.hpp"
#include "CDPL/Pharm/FeatureMapping.hpp"
#include "CDPL/Pharm/FeatureTypeHistogram.hpp"
#include "CDPL/Pharm/Feature3DCoordinatesFunction.hpp"
#include "CDPL/Pharm/FeatureContainerProperty.hpp"
#include "CDPL/Pharm/FeatureProperty.hpp"
#include "CDPL/Pharm/FeatureContainerPropertyDefault.hpp"
#include "CDPL/Pharm/FeaturePropertyDefault.hpp"
#include "CDPL/Pharm/ControlParameter.hpp"
#include "CDPL/Pharm/ControlParameterDefault.hpp"
#include "CDPL/Pharm/DataFormat.hpp"
#include "CDPL/Pharm/FeatureType.hpp"
#include "CDPL/Pharm/FeatureGeometry.hpp"
#include "CDPL/Pharm/FeatureGenerator.hpp"
#include "CDPL/Pharm/PatternBasedFeatureGenerator.hpp"
#include "CDPL/Pharm/AromaticFeatureGenerator.hpp"
#include "CDPL/Pharm/HydrophobicFeatureGenerator.hpp"
#include "CDPL/Pharm/HydrophobicAtomFeatureGenerator.hpp"
#include "CDPL/Pharm/HBondAcceptorFeatureGenerator.hpp"
#include "CDPL/Pharm/HBondDonorFeatureGenerator.hpp"
#include "CDPL/Pharm/PosIonizableFeatureGenerator.hpp"
#include "CDPL/Pharm/NegIonizableFeatureGenerator.hpp"
#include "CDPL/Pharm/XBondAcceptorFeatureGenerator.hpp"
#include "CDPL/Pharm/XBondDonorFeatureGenerator.hpp"
#include "CDPL/Pharm/PharmacophoreGenerator.hpp"
#include "CDPL/Pharm/DefaultPharmacophoreGenerator.hpp"
#include "CDPL/Pharm/InteractionAnalyzer.hpp"
#include "CDPL/Pharm/DefaultInteractionAnalyzer.hpp"
#include "CDPL/Pharm/InteractionConstraintConnector.hpp"
#include "CDPL/Pharm/FeatureDistanceConstraint.hpp"
#include "CDPL/Pharm/HydrophobicInteractionConstraint.hpp"
#include "CDPL/Pharm/IonicInteractionConstraint.hpp"
#include "CDPL/Pharm/OrthogonalPiPiInteractionConstraint.hpp"
#include "CDPL/Pharm/ParallelPiPiInteractionConstraint.hpp"
#include "CDPL/Pharm/CationPiInteractionConstraint.hpp"
#include "CDPL/Pharm/HBondingInteractionConstraint.hpp"
#include "CDPL/Pharm/XBondingInteractionConstraint.hpp"
#include "CDPL/Pharm/InteractionPharmacophoreGenerator.hpp"
#include "CDPL/Pharm/FeatureInteractionScore.hpp"
#include "CDPL/Pharm/FeatureInteractionScoreCombiner.hpp"
#include "CDPL/Pharm/FeatureDistanceScore.hpp"
#include "CDPL/Pharm/HydrophobicInteractionScore.hpp"
#include "CDPL/Pharm/IonicInteractionScore.hpp"
#include "CDPL/Pharm/HBondingInteractionScore.hpp"
#include "CDPL/Pharm/XBondingInteractionScore.hpp"
#include "CDPL/Pharm/OrthogonalPiPiInteractionScore.hpp"
#include "CDPL/Pharm/ParallelPiPiInteractionScore.hpp"
#include "CDPL/Pharm/CationPiInteractionScore.hpp"
#include "CDPL/Pharm/SpatialFeatureMapping.hpp"
#include "CDPL/Pharm/PharmacophoreAlignment.hpp"
#include "CDPL/Pharm/FeatureTypeMatchFunctor.hpp"
#include "CDPL/Pharm/FeaturePairDistanceMatchFunctor.hpp"
#include "CDPL/Pharm/FeaturePositionMatchFunctor.hpp"
#include "CDPL/Pharm/FeatureGeometryMatchFunctor.hpp"
#include "CDPL/Pharm/PharmacophoreFitScore.hpp"
#include "CDPL/Pharm/CDFPharmacophoreInputHandler.hpp"
#include "CDPL/Pharm/PMLPharmacophoreInputHandler.hpp"
#include "CDPL/Pharm/CDFFeatureContainerOutputHandler.hpp"
#include "CDPL/Pharm/PMLFeatureContainerOutputHandler.hpp"
#include "CDPL/Pharm/CDFPharmacophoreReader.hpp"
#include "CDPL/Pharm/PMLPharmacophoreReader.hpp"
#include "CDPL/Pharm/CDFFeatureContainerWriter.hpp"
#include "CDPL/Pharm/PMLFeatureContainerWriter.hpp"
#include "CDPL/Pharm/ScreeningDBCreator.hpp"
#include "CDPL/Pharm/ScreeningDBAccessor.hpp"
#include "CDPL/Pharm/ScreeningProcessor.hpp"
#include "CDPL/Pharm/PharmacophoreFitScreeningScore.hpp"
#include "CDPL/Pharm/MoleculeFunctions.hpp"
#include "CDPL/Pharm/PharmacophoreFunctions.hpp"
#include "CDPL/Pharm/FeatureFunctions.hpp"
#include "CDPL/Pharm/FeatureContainerFunctions.hpp"
#include "CDPL/Pharm/FeatureSetFunctions.hpp"
#include "CDPL/Pharm/ControlParameterFunctions.hpp"
#include "CDPL/Pharm/UtilityFunctions.hpp"
#include "CDPL/Pharm/FileScreeningHitCollector.hpp"
#include "CDPL/Pharm/CDFGZPharmacophoreInputHandler.hpp"
#include "CDPL/Pharm/CDFBZ2PharmacophoreInputHandler.hpp"
#include "CDPL/Pharm/CDFGZFeatureContainerOutputHandler.hpp"
#include "CDPL/Pharm/CDFBZ2FeatureContainerOutputHandler.hpp"
#include "CDPL/Pharm/CDFGZPharmacophoreReader.hpp"
#include "CDPL/Pharm/CDFBZ2PharmacophoreReader.hpp"
#include "CDPL/Pharm/CDFGZFeatureContainerWriter.hpp"
#include "CDPL/Pharm/CDFBZ2FeatureContainerWriter.hpp"
#include "CDPL/Pharm/PharmacophoreReader.hpp"
#include "CDPL/Pharm/FeatureContainerWriter.hpp"
# include "CDPL/Pharm/PSDPharmacophoreInputHandler.hpp"
# include "CDPL/Pharm/PSDMoleculeInputHandler.hpp"
# include "CDPL/Pharm/PSDMolecularGraphOutputHandler.hpp"
# include "CDPL/Pharm/PSDPharmacophoreReader.hpp"
# include "CDPL/Pharm/PSDMoleculeReader.hpp"
# include "CDPL/Pharm/PSDMolecularGraphWriter.hpp"
# include "CDPL/Pharm/PSDScreeningDBCreator.hpp"
# include "CDPL/Pharm/PSDScreeningDBAccessor.hpp"

#endif // CDPL_PHARM_HPP
