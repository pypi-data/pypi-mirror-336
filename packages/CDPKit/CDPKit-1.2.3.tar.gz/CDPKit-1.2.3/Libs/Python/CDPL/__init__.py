##
# __init__.py  
#
# This file is part of the Chemical Data Processing Toolkit
#
# Copyright (C) 2003 Thomas Seidel <thomas.seidel@univie.ac.at>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; either
# version 2 of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with this program; see the file COPYING. If not, write to
# the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
# Boston, MA 02111-1307, USA.
##


from __future__ import absolute_import

import os
import sys
import platform

if sys.version_info[1] >= 8 and platform.system() == 'Windows':
    if 'site-packages' in __file__:
        os.add_dll_directory(os.path.dirname(__file__))
    else:
        os.add_dll_directory(os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), '@CDPKIT_EXECUTABLE_INSTALL_DIR@'))

from ._cdpl import *

__version__ = VERSION_STRING
