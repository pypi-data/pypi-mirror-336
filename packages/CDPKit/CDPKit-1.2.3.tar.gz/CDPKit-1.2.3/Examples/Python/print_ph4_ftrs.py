#!/bin/env python

##
# print_ph4_ftrs.py 
#
# This file is part of the Pharmical Data Processing Toolkit
#
# Copyright (C) 2003 Thomas Seidel <thomas.seidel@univie.ac.at>
#
# Permission to use, copy, modify, and/or distribute this software for any purpose with
# or without fee is hereby granted.
# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD
# TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN
# NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL
# DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER
# IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
# CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
##


import sys

import CDPL.Pharm as Pharm
import CDPL.Chem as Chem


# outputs all (available) properties of the features stored in the given feature container
def outputFeatureProps(ph4: Pharm.FeatureContainer) -> None: 
    ftr_type_str = { Pharm.FeatureType.UNKNOWN               : 'UNKNOWN',
                     Pharm.FeatureType.HYDROPHOBIC           : 'HYDROPHOBIC',
                     Pharm.FeatureType.AROMATIC              : 'AROMATIC',
                     Pharm.FeatureType.NEGATIVE_IONIZABLE    : 'NEGATIVE_IONIZABLE',
                     Pharm.FeatureType.POSITIVE_IONIZABLE    : 'POSITIVE_IONIZABLE',
                     Pharm.FeatureType.H_BOND_DONOR          : 'H_BOND_DONOR',
                     Pharm.FeatureType.H_BOND_ACCEPTOR       : 'H_BOND_ACCEPTOR',
                     Pharm.FeatureType.HALOGEN_BOND_DONOR    : 'HALOGEN_BOND_DONOR',
                     Pharm.FeatureType.HALOGEN_BOND_ACCEPTOR : 'HALOGEN_BOND_ACCEPTOR',
                     Pharm.FeatureType.EXCLUSION_VOLUME      : 'EXCLUSION_VOLUME' }
  
    geom_str = { Pharm.FeatureGeometry.UNDEF   : 'UNDEF',
                 Pharm.FeatureGeometry.SPHERE  : 'SPHERE',
                 Pharm.FeatureGeometry.VECTOR  : 'VECTOR',
                 Pharm.FeatureGeometry.PLANE   : 'PLANE' }

    print('Composition of pharmacophore \'%s\':' % Pharm.getName(ph4))

    for i in range(0, len(ph4)):
        ftr = ph4[i]

        print(' - Feature #%s:' % str(i))
        print('  - Type: %s' % ftr_type_str[Pharm.getType(ftr)])
        print('  - Geometry: %s' % geom_str[Pharm.getGeometry(ftr)])
        print('  - Tolerance: %s' % Pharm.getTolerance(ftr))
        print('  - Weight: %s' % Pharm.getWeight(ftr))
        print('  - Optional: %s' % Pharm.getOptionalFlag(ftr))
        print('  - Disabled: %s' % Pharm.getDisabledFlag(ftr))
        print('  - Length: %s' % Pharm.getLength(ftr))
        print('  - Hydrophobicity: %s' % Pharm.getHydrophobicity(ftr))

        if Chem.has3DCoordinates(ftr):         # Pharm.Feature derives from Chem.Entity3D - therefore a function from the Chem package is used here!
            print('  - Position: %s' % Chem.get3DCoordinates(ftr))
 
        if Pharm.hasOrientation(ftr):
            print('  - Orientation: %s' % Pharm.getOrientation(ftr))

def main() -> None:
    if len(sys.argv) < 2:
        sys.exit('Usage: %s <input pharm. file>' % sys.argv[0])

    # create reader for input pharmacophores (format specified by file extension)
    reader = Pharm.PharmacophoreReader(sys.argv[1]) 
    
    # create an instance of the default implementation of the Pharm.Pharmacophore interface
    ph4 = Pharm.BasicPharmacophore()
    
    # read and process pharmacophores one after the other until the end of input has been reached
    try:
        while reader.read(ph4): 
            try:
                outputFeatureProps(ph4)
            except Exception as e:
                sys.exit('Error: processing of pharmacophore failed: ' + str(e))
                
    except Exception as e: # handle exception raised in case of severe read errors
        sys.exit('Error: reading pharmacophore failed: ' + str(e))

    sys.exit(0)
        
if __name__ == '__main__':
    main()
