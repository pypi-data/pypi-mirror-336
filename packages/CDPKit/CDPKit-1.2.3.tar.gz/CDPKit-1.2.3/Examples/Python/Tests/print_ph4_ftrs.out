Composition of pharmacophore '1dwc_MIT_1_H_1__D___':
 - Feature #0:
  - Type: HYDROPHOBIC
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 2.3680062294006348
  - Position: [3](30.6431,10.208,24.6583)
 - Feature #1:
  - Type: HYDROPHOBIC
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 2.0086073875427246
  - Position: [3](31.8824,10.8049,27.1081)
 - Feature #2:
  - Type: HYDROPHOBIC
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.8417263627052307
  - Position: [3](33.794,10.424,28.673)
 - Feature #3:
  - Type: HYDROPHOBIC
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 1.134719967842102
  - Position: [3](31.1545,17.8945,24.2796)
 - Feature #4:
  - Type: HYDROPHOBIC
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.8417263627052307
  - Position: [3](32.8,10.219,20.483)
 - Feature #5:
  - Type: AROMATIC
  - Geometry: PLANE
  - Tolerance: 0.8999999761581421
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](30.3358,10.8502,25.0028)
  - Orientation: [3](0.688932,0.570169,-0.447527)
 - Feature #6:
  - Type: NEGATIVE_IONIZABLE
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](32.2127,14.984,20.5903)
 - Feature #7:
  - Type: POSITIVE_IONIZABLE
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](32.8338,19.8455,25.3797)
 - Feature #8:
  - Type: H_BOND_DONOR
  - Geometry: VECTOR
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 2.5
  - Hydrophobicity: 0.0
  - Position: [3](30.826,12.247,27.3)
  - Orientation: [3](-0.525068,0.845075,0.100756)
 - Feature #9:
  - Type: H_BOND_DONOR
  - Geometry: VECTOR
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 2.5
  - Hydrophobicity: 0.0
  - Position: [3](29.732,14.82,25.183)
  - Orientation: [3](0.142949,0.843986,0.516966)
 - Feature #10:
  - Type: H_BOND_DONOR
  - Geometry: VECTOR
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 2.5
  - Hydrophobicity: 0.0
  - Position: [3](31.58,19.501,25.067)
  - Orientation: [3](-0.727735,0.427255,0.536521)
 - Feature #11:
  - Type: H_BOND_DONOR
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](33.869,19.302,24.687)
  - Orientation: [3](0.758761,-0.423905,-0.494556)
 - Feature #12:
  - Type: H_BOND_DONOR
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](33.048,20.701,26.406)
  - Orientation: [3](0.155763,0.610444,0.776592)
 - Feature #13:
  - Type: H_BOND_ACCEPTOR
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](33.869,19.302,24.687)
  - Orientation: [3](-0.758761,0.423905,0.494556)
 - Feature #14:
  - Type: H_BOND_ACCEPTOR
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](31.58,19.501,25.067)
  - Orientation: [3](0.727735,-0.427255,-0.536521)
 - Feature #15:
  - Type: H_BOND_ACCEPTOR
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](32.518,15.723,21.305)
  - Orientation: [3](-0.228313,-0.885953,-0.403684)
 - Feature #16:
  - Type: H_BOND_ACCEPTOR
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](28.554,13.649,27.057)
  - Orientation: [3](0.137578,-0.0944978,-0.985973)
 - Feature #17:
  - Type: H_BOND_ACCEPTOR
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](27.669,13.399,24.68)
  - Orientation: [3](0.746693,0.0785992,0.660509)
 - Feature #18:
  - Type: H_BOND_ACCEPTOR
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](32.52,14.296,25.111)
  - Orientation: [3](-0.565528,-0.118711,-0.816141)
 - Feature #19:
  - Type: H_BOND_ACCEPTOR
  - Geometry: SPHERE
  - Tolerance: 1.5
  - Weight: 1.0
  - Optional: False
  - Disabled: False
  - Length: 1.0
  - Hydrophobicity: 0.0
  - Position: [3](31.878,14.577,19.649)
  - Orientation: [3](0.296973,0.0611894,0.952923)
